app/emulator/include/emulator.hpp

```diff
@@ -17,7 +17,7 @@
 
 #include <iostream>
 #include <string>
-#include "../../libcanio/include/canio.hpp"
+#include "../../../lib/libcanio/include/canio.hpp"
```

Comment:
use #include canio.hpp ( I think the folder already is included in cmake, otherwise add it there )